docs/tutorials/annotation/annotate_points.md

```diff
@@ -4,8 +4,8 @@
 
 ```{Admonition} DEPRECATED ATTRIBUTES
 :class: warning
-As of napari 0.5.0, `edge_*` attributes are being renamed to 
-`border_*` attributes. We have yet to update the images and/or videos in 
+As of napari 0.5.0, `edge_*` attributes are being renamed to
```

Comment:
The changes in this file mostly come from the corresponding PR over on the pointannotator repo: https://github.com/kevinyamauchi/PointAnnotator/pull/6

There are also some other name swaps.